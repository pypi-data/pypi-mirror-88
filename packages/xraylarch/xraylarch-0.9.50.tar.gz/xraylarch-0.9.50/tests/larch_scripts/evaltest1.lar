1+2*3                    
-pi/3                    
2*(4/3)                  
2*4/3                    
2*(4+3)                  
2*4+3                    
5/2+3                    
5/2*3                    
6/2/3                    
2**3+3                    
2**(3+3)                  
2**(3*3)                  
+3                       
3 * -2                   
-3**2                     
2**3**2                    
2**4 + 1                  
8*3**2 + 2                
-1**3                    
cos(0.1)                 
sin(0.1)                 
sin(pi/2 - tan(0.1))     
tan(0.2 + 1.0j)          
pi
e**(1j*pi) + 1      
