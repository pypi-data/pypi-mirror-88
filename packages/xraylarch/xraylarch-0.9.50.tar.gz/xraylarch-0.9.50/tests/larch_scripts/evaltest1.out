7
-1.0471975512
2.66666666667
2.66666666667
14
11
5.5
7.5
1.0
11
64
512
3
-6
-9
512
17
74
-1
0.995004165278
0.0998334166468
0.994970698098
(0.0831511848472+0.774431266899j)
3.14159265359
1.22460635382e-16j
